src/audio/module_adapter/Kconfig

```diff
@@ -192,7 +148,53 @@ if CADENCE_CODEC
 		  This option is a string and takes the full name of the SRC library binary.
 	endif
 
-endif
+endif # Cadence
+
+	config COMP_VOLUME
+		bool "Volume component"
+		default y
+		depends on COMP_MODULE_ADAPTER
+		help
+		  Select for Volume component
+
+
```

Comment:
Had to push to Github to catch this one; expect a force push where I fix it by myself.